tests/src/page_objects/locators/locators.py

```diff
@@ -55,3 +55,4 @@
 DownloadLogsButton = '//span[contains(text(),"Download log")]'
 ExtractionLangFilterOption = '//*[@id="extractionLanguageFilterOptions"]'
 CollisionResolutionOption = 'collisionResolutionOptions'
+OpeningFileText = '//pre[contains(text(),"file_4MB.exe")]'
```

Comment:
I think that locators.py shouldn't contain test specific names. In this case OpeningFileText should be defined as test variable since it can't be use by any other test that will use different file.

Reply:
Agreed. Done